include/cuco/static_set.cuh

```diff
@@ -86,8 +86,8 @@ template <class Key,
           cuda::thread_scope Scope = cuda::thread_scope_device,
           class KeyEqual           = thrust::equal_to<Key>,
           class ProbingScheme      = experimental::double_hashing<4,  // CG size
-                                                             cuco::murmurhash3_32<Key>,
-                                                             cuco::murmurhash3_32<Key>>,
+                                                             cuco::xxhash_32<Key>,
+                                                             cuco::xxhash_32<Key>>,
```

Comment:
A separate PR for this change per the previous discussions.

Reply:
I see, I thought this was just targeting the pre-refactor structs. Will do.